packages/manager/apps/dedicated/client/app/app.bundle.js

```diff
@@ -13,8 +13,8 @@ import './dedicated/dedicated-server/servers/bandwidth/private-order/private-ord
 import './dedicated/dedicated-server/servers/bandwidth/public-order/public-order.module';
 import './dedicated/dedicated-server/servers/interfaces/attach/interfaces-attach.module';
 import './dedicated/dedicated-server/servers/interfaces/detach/interfaces-detach.module';
-import './dedicated/dedicated-server/servers/interfaces/ola-activation/ola-activation.module';
-import './dedicated/dedicated-server/servers/interfaces/ola-configuration/ola-configuration.module';
+// import './dedicated/dedicated-server/servers/interfaces/ola-activation/ola-activation.module';
```

Comment:
**Note** : Temporary, but what's the goal of that file ?

Reply:
No need anymore because you execute the module by the index.js file